03.bowling/bowling.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+point = 0
+
+if frames.size > 10
+  first_strike = frames[9][0] == 10
+  second_strike = frames[10][0] == 10
+
+  last_frame =
+    if first_strike && second_strike
+      [10, 10, frames[11][0]]
+    elsif first_strike
+      [10, frames[10][0], frames[10][1]]
+    else
+      [frames[9][0], frames[9][1], frames[10][0]]
```

Comment:
```suggestion
      [10, *frames[10]]
    else
      [*frames[9], frames[10][0]]
```

としてもいいかもしれませんね。
ただ変更前の方がわかりやすい、というメリットもあるので好みで選んでもらって大丈夫です